### DISCUSSION TITLE
Selected department in widget

### DISCUSSION THREAD

**Discussion by Erika06** [LiveHelperChat/livehelperchat#2065] [General]
When I generate the embed code I select a department but on the page where the user is going to start the Chat that department is not selected, but instead all the departments are shown.

I clarify that I have it configured so that if there are no operators online an email is sent.

**Comment by remdex** [comment]
It would be helpfull if you included your embed code and screenshot. From description it's just unclear. You can remove domain from your embed code just.

**Reply by Erika06** [reply]
Thank you for your response

_<script type="text/javascript">
var LHCChatOptions = {};
LHCChatOptions.opt = {widget_height:340,widget_width:300,popup_height:520,popup_width:500};
(function() {
var po = document.createElement('script'); po.type = 'text/javascript'; po.async = true;
var referrer = (document.referrer) ? encodeURIComponent(document.referrer.substr(document.referrer.indexOf('://')+1)) : '';
var location  = (document.location) ? encodeURIComponent(window.location.href.substring(window.location.protocol.length)) : '';
po.src = '//148.210.21.141/lhcweb/index.php/chat/getstatus/(click)/internal/(position)/bottom_right/(ma)/br/(top)/350/(units)/pixels/(leaveamessage)/true/(department)/1?r='+referrer+'&l='+location;
var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(po, s);
})();
</script>_
![lhcweb-D1](https://github.com/LiveHelperChat/livehelperchat/assets/28407953/2b8fdae5-c159-4b02-9e06-163b023759b6)


We need the departments to be separated to be able to send email when it is not available even if there are operators in the other department

thanks!

**Reply by remdex** [reply]
You are using old embed code. Use new

**Reply by Erika06** [reply]
Thank you.
I used the new Widget code and looks fine.
Now I have that same problem with the embed code.

How can I generate de new embed code?
